Buildings/Controls/OBC/FDE/DOAS/DOAScontroller.mo

```diff
@@ -0,0 +1,494 @@
+within Buildings.Controls.OBC.FDE.DOAS;
+block DOAScontroller "DOAS controller built from DOAS blocks."
+
+  parameter Boolean vvUnit = true
+    "Set true when unit serves variable volume system."
+    annotation(Dialog(tab = "DOAS", group = "General"));
+
+  parameter Real damSet(
```

Comment:
Rename to damSetThr. Comment should be "Damper position threshold above which the duct static pressure setpoint is reset"